jazz-build-module/environment-deployment-metadata-loader.groovy

```diff
@@ -60,6 +60,9 @@ def getEnvironmentLogicalId() {
 		}
 		g_environment_logical_id = environment_logical_id
 	}
+	if(g_service_config['domain'] == "jazz"){
```

Comment:
Indentation "if (" here and in other files

Reply:
fixed

Reply:
fixed